webtool/test_llm.py

```
@@ -433,3 +433,56 @@ def test_nicht_dekodierbare_eingabedatei_wird_zur_llmerror(tmp_path):
+        gesehen["context"] = context
+        return DummyResponse()
+
+    monkeypatch.setattr(llm.urllib.request, "urlopen", fake_urlopen)
+    res = llm._request("https://example.com/api", {"header": "val"})
+    assert res == {"ok": True}
+    assert gesehen["context"] is not None
+    import ssl
+    assert isinstance(gesehen["context"], ssl.SSLContext)
+
+
+def test_google_provider_hat_default_model(cfg):
```

Ruff (0.16.1)

[warning] Unused function argument: `cfg`

(ARG001)